apps/frontend/src/libs/components/proggress-bar/progress-bar.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+import CheckIcon from "~/assets/img/check-icon.svg?react";
+import { getValidClassNames } from "~/libs/helpers/helpers.js";
+
+import { type Step } from "./libs/types/types.js";
+import styles from "./styles.module.css";
+
+const LAST_INDEX_OFFSET = 1;
+
+type Properties = {
+	steps: Step[];
+};
+
+const ProgressBar: React.FC<Properties> = ({ steps }) => {
+	return (
+		<div className={styles["progress-bar"]}>
+			{steps.map((stepObject, index) => {
+				const isLastStep = index === steps.length - LAST_INDEX_OFFSET;
```

Comment:
I don't think we need the `LAST_INDEX_OFFSET` constant. The last one will always be `.length - 1`. Or could there be a case when we change `LAST_INDEX_OFFSET` to something not `1`?

Reply:
I chose not to use the number 1 directly to avoid violating the(ESLint: No magic number: 1.(@typescript-eslint/ no-magic-numbers)), which encourages avoiding "magic numbers" in code.
Maybe not the best approach but if we need such constant globally I'll change.